src/tqec/templates/atomic/__init__.py

```diff
@@ -0,0 +1,16 @@
+from .rectangle import (
+    AlternatingRectangleTemplate,
+    RawRectangleTemplate,
+)
+
+from .square import (
+    AlternatingSquareTemplate,
+    AlternatingCornerSquareTemplate,
+)
+
+__all__ = [
+    "AlternatingRectangleTemplate",
+    "RawRectangleTemplate",
+    "AlternatingSquareTemplate",
+    "AlternatingCornerSquareTemplate",
```

Comment:
The `AlternatingCornerSquareTemplate` is currently on hold. If I remember correctly, the implementation uses permutations and I found at least one issue with that approach, as noted in the docstring:
https://github.com/QCHackers/tqec/blob/c1bf3d44697b77f04be9ee36436dca606d07b505/src/tqec/templates/atomic/square.py#L60-L63
I should have opened an issue back then, it seems like I forgot, I will do that just after finishing this review.
In the meantime, I am in favour of commenting out this class from the top-level imports, as a weak sign that it is not yet ready for the public API.

Reply:
```suggestion
```